site/content/en/docs/Installation/_index.md

```diff
@@ -6,11 +6,11 @@ description: >
   Follow this guide to setup Open Match in your Kubernetes cluster.
 ---
 
-In this quickstart, we'll create a Kubernetes cluster, install Open Match, and create matches with the example tools.
+In this quickstart, we'll create a Kubernetes cluster and install Open Match.
 
 ## Setup Kubernetes Cluster
 
-This guide is for users that do not have a Kubernetes cluster. If you already have one that you can install Open Match into, skip this section.
+This guide is for users that do not have a Kubernetes cluster. If you already have one that you can install Open Match into, see [how to get your first match with Open Match]({{< ref "/docs/Getting Started" >}}).
```

Comment:
What this comment is saying is that you can skip the setup k8s cluster section (note the number of #s, more means a nested section under fewer).  They still need to install open match itself.